src/server/typingsInstaller/typingsInstaller.ts

```diff
@@ -378,8 +380,8 @@ namespace ts.server.typingsInstaller {
                 this.projectWatchers.set(projectName, watchers);
             }
 
+            watchers.isInvoked = false;

```

Comment:
so were we having same watchers in multiple projects? i am not sure i understand the change here..

Reply:
Previously the isInvoked was local variable that was captured. If there were additional watches added to the project (because it has more files etc) the new isInvoked would be used for additional watchers, but the existing watches would still refer to the previous isInvoked thus invalidating the typing multiple times or ignoring the typings assuming it was already invalidating.